src/utils/format.js

```diff
@@ -26,6 +26,9 @@ export function formatUptimeDuration (
     return defaultValue
   }
 
-  const formatted: string = moment.duration(durationInMS).format('d [days], h:mm')
+  // (up for 0 days, 0 hours, 1 minute, 37 seconds)
+  const formatted: string = moment.duration(durationInMS).format(
```

Comment:
I think it is better to shut off seconds after 2 minutes as @gregsheremeta mentioned in issue https://github.com/oVirt/ovirt-web-ui/issues/893#issuecomment-447343729

Reply:
Fixed